__tests__/main.test.js

```diff
@@ -36,3 +36,35 @@ describe('API', () => {
 			})
 	})
 })
+describe('Articles', () => {
+	it('GET:200 responds with the matching id by given unique id', () => {
+		return request(app)
+			.get('/api/articles/1')
+			.expect(200)
+			.then(({ body: { article } }) => {
+				expect(typeof article.article_id).toBe('number')
```

Comment:
We could /should assert the actual number here as we have made a request for a specific article.